### PR TITLE
add repr infomation for layer_norm and rms_norm

### PR DESCRIPTION
**Pull request by wizyoung**
## Summary
Add repr information for layernorm and rmsnorm class so that the useful layer information can be displayed after the model is printed. Other classes are not modified because they inherit from related torch.nn classes, or there are torch.nn sub-modules.

## Testing Done

- Hardware Type: <BLANK>
- [x] run `make test` to ensure correctness
- [x] run `make checkstyle` to ensure code style
- [x] run `make test-convergence` to ensure convergence

### CODE REVIEW

File: src/liger_kernel/transformers/layer_norm.py [hunk 1]
```diff
@@ -23,3 +25,6 @@ def forward(self, hidden_states):
         return LigerLayerNormFunction.apply(
             hidden_states, self.weight, self.bias, self.variance_epsilon
         )
+        
+    def extra_repr(self) -> str:
+        return f'{self.hidden_size}, eps={self.eps}'
```
Comment (ByronHsu) [inline src/liger_kernel/transformers/layer_norm.py:30]: why no weight here?

Comment (wizyoung) [reply src/liger_kernel/transformers/layer_norm.py:30]: This is in line with nn.LayerNorm whose extra_repr is:
```python
def extra_repr(self) -> str:
        return '{normalized_shape}, eps={eps}, ' \
            'elementwise_affine={elementwise_affine}'.format(**self.__dict__)
```
For rms_norm, it is in line with huggingface transformers.

Comment (wizyoung) [reply src/liger_kernel/transformers/layer_norm.py:30]: Do we need to keep them consistent here?

Comment (ByronHsu) [reply src/liger_kernel/transformers/layer_norm.py:30]: oh sg

**Review by ByronHsu — APPROVED** [review]

**Comment by ByronHsu** [comment]
need to fix the checkstyle

**Comment by wizyoung** [comment]
now checkstyle passed

**Review by ByronHsu — APPROVED** [review]

**Pull request merged**